packages/api-bindings/codegen-api.yml

```diff
@@ -56,7 +56,7 @@ generates:
         TxId: string
         UnixTimestamp: number
         URI: string
-        URL: Url
+        URL: string
```

Comment:
Should `URI` and `URL` now be mapped to the `URI` and `URL` branded types in shared-kernel?

Reply:
right, will do!

Reply:
done